presto-hive/src/test/java/io/prestosql/plugin/hive/util/TestHiveUtil.java

```diff
@@ -75,6 +83,29 @@ public void testToPartitionValues()
         assertToPartitionValues("pk=__HIVE_DEFAULT_PARTITION__");
     }
 
+    @Test
+    public void testGetInputFormat()
+    {
+        Configuration configuration = new Configuration(false);
+
+        Properties avroSymlinkSchema = new Properties();
+        avroSymlinkSchema.setProperty(FILE_INPUT_FORMAT, SymlinkTextInputFormat.class.getName());
+        avroSymlinkSchema.setProperty(SERIALIZATION_LIB, AVRO.getSerDe());
+        assertTrue(getInputFormat(configuration, avroSymlinkSchema, false) instanceof SymlinkTextInputFormat);
```

Comment:
Use `assertInstanceOf()` since it provides a better error message. In general, `assertTrue()` should not be used with a predicate -- try to express the predicate using an assertion method or AssertJ.